index.ts

```diff
@@ -9,26 +8,29 @@ import {MongoConnection} from "./api/db/MongoConnection";
 import {RedisConnection} from "./api/db/RedisConnection";
 import volume from "./api/v1/routes/volume";
 
-// Connect to DBs
-async () => {
+async function setupDBs() {
```

Comment:
Can't find a cleaner way to bootstrap the API only after the DBs have been created. async/await in root isn't allowed in typescript here

Reply:
I think bootstrapping services is fine here